stdlib/private/StdlibUnittest/SymbolLookup.swift

```diff
@@ -23,7 +23,7 @@
 
 #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
   let RTLD_DEFAULT = UnsafeMutableRawPointer(bitPattern: -2)
-#elseif os(Linux)
+#elseif os(Linux) || os(WASI)
   let RTLD_DEFAULT = UnsafeMutableRawPointer(bitPattern: 0)
```

Comment:
WebAssembly does not have dynamic linking, does it? If it does not, then `RTLD_DEFAULT` is not meaningful I think.